ecpp

```diff
@@ -112,7 +150,114 @@ def main():
     except KeyboardInterrupt:
         print(n)
         raise
-    print("Number is prime")
+    print("\n")
+    print("[+] number is prime\n")
+
+def correlate(moduli, certs=None, verify_opt=None, primo_opt=None):
+    """Correlate between functionalities"""
+    missing_cert_flag = False
+    cert_files = []
+    if certs is None:
+        certs = certs_dir
+    for path, _, files in os.walk(certs):
+        for name in files:
+            cert_files.append(os.path.join(path, name))
+    status = None
+    cert_dict = {}
+    for cert in cert_files:
+        if cert.endswith('.out'):
+            path = cert
+            cert_candidate = candidate_finder(path)
+            cert_dict[cert_candidate] = path
+    with open(str(moduli)) as f:
+        inputfile = filter(None, (line.rstrip() for line in f))
+        for count_line, line in enumerate(inputfile, 1):
+            if line.startswith('#'):
+                continue
+            prime_moduli = (line.split(" ")[6])
+            int_prime_moduli = int(prime_moduli, 16)
```

Comment:
the code also should check for `(int_prime_moduli - 1) / 2` to verify that the number is not only prime but a safe prime